ext/web/lib.deno_web.d.ts

```diff
@@ -1111,7 +1111,7 @@ declare var MessageEvent: {
 };
 
 /** @category Events */
-type Transferable = MessagePort | ArrayBuffer;
+type Transferable = MessagePort | ArrayBufferLike;
```

Comment:
A `SharedArrayBuffer` `[[Serializable]]` but not `[[Transferable]].
https://github.com/microsoft/TypeScript/blob/0652664e6408f7704395fc53fe245e2e77367fe8/src/lib/dom.generated.d.ts#L29370

Reply:
Hmmm... i don't remember making this change. Thanks!